src/main/endpoints.cljs

```diff
@@ -8,11 +8,18 @@
             [taoensso.timbre :as log]
             [com.rpl.specter :as s]
             [cljs.nodejs :as nodejs]
-            [util.os :as os]
-            [express.web-api :as web]
+            [bidi.bidi :refer [path-for]]
+            [fast-twitch.nav :refer [cached-routes]]
+            [fast-twitch.os :as os]
+            [fast-twitch.web-api :as web]
             [ui.components.core :as cmpt]
             [services.core :as svc]))
 
+(defn redirect [route-key]
```

Comment:
Isn't already defined in FastTwitch?

Reply:
Yea it is broken so I was using this until you merge pull requet